src/css/components/_progress.scss

```diff
@@ -58,6 +58,19 @@
   }
 }
 
+// Current Time "tooltip"
+.video-js .vjs-progress-control:hover .vjs-play-progress:after {
```

Comment:
I wonder whether it'll be better to separate showing this control on hover from styling the control.